api/mock_history_data.json

```diff
@@ -0,0 +1,60 @@
+{
+    "": [
+      {
+        "date": "167252000",
```

Comment:
The date should represent a unix timestamp, so I think this date is too short, and might represent seconds rather than milliseconds.